example_project_bzl_4/third_party/readline/defs.bzl

```diff
@@ -0,0 +1,17 @@
+load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl", "nixpkgs_package")
+load("//third_party/nix:defs.bzl", "NIX_REPOSITORIES")
+
+def readline():
+    nixpkgs_package(
+        name = "readline.headers",
+        attribute_path = "readline81.dev",
+        build_file = "//third_party/readline:BUILD.bazel.tmpl",
+        repositories = NIX_REPOSITORIES,
+    )
+
+    nixpkgs_package(
+        name = "readline.lib",
+        attribute_path = "readline81.out",
```

Comment:
`readline81` would have also sufficed, but I want to explore what's the thing with using `readline81.out`.